heron/packing/src/java/com/twitter/heron/packing/roundrobin/ResourceCompliantRRPacking.java

```diff
@@ -146,7 +147,7 @@ public PackingPlan pack() {
     Map<String, Long> ramMap = TopologyUtils.getComponentRamMapConfig(topology);
 
     List<TopologyAPI.Config.KeyValue> topologyConfig = topology.getTopologyConfig().getKvsList();
-    int paddingPercentage = TopologyUtils.getConfigWithDefault(
+    this.paddingPercentage = TopologyUtils.getConfigWithDefault(
```

Comment:
better to set all class member values during initialization, instead of as a side effect from a logic method.